metrics/mixin/rules/rules.libsonnet

```diff
@@ -36,7 +36,6 @@
       },
       {
         name: 'ODF_standardized_metrics.rules',
-        interval: '30s',
```

Comment:
Why do we need to remove this interval?

Reply:
The BZ for which this PR is initiated is actually a re-opened BZ.
The line which is removed here was the previous fix which was not effective (and QE had to re-open the BZ again) and it is not needed now